data_steward/validation/email_notification.py

```diff
@@ -11,17 +11,63 @@
 import mandrill
 from jinja2 import Template
 from matplotlib import image as mpimg
+import requests
 
 # Project imports
 import app_identity
-from constants.utils import bq as bq_consts
+import bq_utils
+from constants.utils import bq as bq_client_consts
+from constants import bq_utils as bq_rest_consts
 from constants.validation import email_notification as consts
 from resources import achilles_images_path
 
 LOGGER = logging.getLogger(__name__)
 
 CONTACT_QUERY_TMPL = Template(consts.CONTACT_LIST_QUERY)
 
+METADATA_URL = 'http://metadata.google.internal/computeMetadata/v1/'
+METADATA_HEADERS = {'Metadata-Flavor': 'Google'}
+SERVICE_ACCOUNT = 'default'
+SCOPES = "https://www.googleapis.com/auth/drive,https://www.googleapis.com/auth/cloud-platform"
+
+
+def get_access_token():
+    url = f'{METADATA_URL}instance/service-accounts/{SERVICE_ACCOUNT}/token?scopes={SCOPES}'
+    # Request an access token from the metadata server.
+    r = requests.get(url, headers=METADATA_HEADERS)
+    r.raise_for_status()
+    # Extract the access token from the response.
+    access_token = r.json()['access_token']
+    return access_token
```

Comment:
Glad you figured this out! I recommend relocating this to a different package `utils.auth` and refactoring `get_access_token()` so that it
1. receives the requested scope(s) as a parameter `scope: List[str]` and 
1. has a docstring 😄